packages/core/src/button/base-button.element.scss

```diff
@@ -146,7 +146,7 @@
 }
 
 :host([status='inverse']) {
-  --color: #{$cds-global-typography-color-100};
+  --color: #{$cds-global-color-white};
```

Comment:
Putting a hold on this for now until we can talk through implications discussed in #5851